packages/appsync-modelgen-plugin/src/preset.ts

```diff
@@ -217,6 +217,37 @@ const generateDartPreset = (
   return config;
 };
 
+const generateManyToManyModelStubs = (options: Types.PresetFnArgs<AppSyncModelCodeGenPresetConfig>) : TypeDefinitionNode[] => {
+  let models = new Array<TypeDefinitionNode>();
+  let manyToManySet = new Set<string>();
+  options.schema.definitions.forEach(def => {
+    if (def.kind === 'ObjectTypeDefinition') {
+      def?.fields?.forEach(field => {
+        field?.directives?.forEach(dir => {
+          if (dir?.name?.value === 'manyToMany') {
+            //manyToManySet.add(dir?.arguments?.relationName);
+            dir?.arguments?.forEach(arg => {
+              if(arg.name.value === 'relationName' && arg.value.kind === 'StringValue') {
```

Comment:
nit: One question about the relationName. Is first letter capitalization required for this field? Since we are generating a new type and the convention for type is first letter capitalized(or PascalCase). Now we are using the raw name directly for intermediate type.

Reply:
fixed